source/convertSchemaToInterface.ts

```diff
@@ -124,9 +125,9 @@ function convertPropertiesToTypes(
       );
     }
     // If neither type or $ref is defined, we can't generate a type. Log an error
-    if (!("type" in value) && !("$ref" in value)) {
+    if (!("type" in value) && !("$ref" in value) && !("alias_for" in value)) {
```

Comment:
What's this change for?

Reply:
Actually, I was hoping you could tell me, and if it's correctly made.

Without the code, the test for the full schema fails. If I recall correctly, something about a type property missing or something on some of the schemas.

Reply:
I just tried removing it now. The test fails with:

```
FAIL  source/emit.test.ts > default ftrack schema
AssertionError: expected [ Array(1) ] to deeply equal []

- Expected
+ Received

- Array []
+ Array [
+   "No type or $ref defined for property user_id in schema UserView",
+ ]
```

Reply:
I patched the file and removed the `alias_for` check

Reply:
Thank you for that!